framework/set/resources/setKeyPath.go

```diff
@@ -13,7 +19,12 @@ func SetKeyPath() string {
 		return ""
 	}
 
-	keyPath = filepath.Join(userDir, "go/src/github.com/rancher/tfp-automation/modules/rancher")
+	mainTfDirPath := os.Getenv(mainTfKeyPath)
+	if mainTfDirPath == "" {
+		logrus.Fatal("Expected env var not set TFFILE_KEY_PATH")
```

Comment:
Just to be consistent with what you have below, can you have this be `logrus.Fatalf`?